docx/oxml/section.py

```diff
@@ -41,22 +45,24 @@ class CT_SectPr(BaseOxmlElement):
     """
     ``<w:sectPr>`` element, the container element for section properties.
     """
-    __child_sequence__ = (
-        'w:footnotePr', 'w:endnotePr', 'w:type', 'w:pgSz', 'w:pgMar',
-        'w:paperSrc', 'w:pgBorders', 'w:lnNumType', 'w:pgNumType', 'w:cols',
-        'w:formProt', 'w:vAlign', 'w:noEndnote', 'w:titlePg',
+    _tag_seq = (
+        'w:headerReference', 'w:footnotePr', 'w:endnotePr', 'w:type', 'w:pgSz',
+        'w:pgMar', 'w:paperSrc', 'w:pgBorders', 'w:lnNumType', 'w:pgNumType',
+        'w:cols', 'w:formProt', 'w:vAlign', 'w:noEndnote', 'w:titlePg',
```

Comment:
* ZeroOrMore is alphabetically prior to ZeroOrOne (on import line)
* That import Line extends beyond 77, wrapped
* When a _tag_seq is added (or touched), all defined elements are included. Added w:footerReference. Usually this is just a legacy issue from back when I was developing that convention :)
* The _tag_seq usage is substantially more elegant now, check the merged commit. That double child_sequence bit must have been one of my earliest trials as I was developing the convention because it is butt-ugly :)